recipes/altair_pandas/meta.yaml

```diff
@@ -0,0 +1,49 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "altair_pandas" %}
+{% set version = "0.1.0dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/altair-viz/{{ name }}/archive/master.tar.gz
```

Comment:
We can only package this once there has been a release.

Reply:
Ok, I can leave this open for now? Or should I close it and wait for the release?

Reply:
You can leave it open, I justed wanted to add a ❌ so that we don't accidentially merge.

Reply:
Sounds good; thanks!